### DISCUSSION TITLE
动态点开评论后的条形广告屏蔽

### DISCUSSION THREAD

**Discussion by pinaZC** [the1812/Bilibili-Evolved#4372] [功能建议 / Ideas]
下图红框中广告，好像只有比较大的活动时才有的广告
用adgrand插件只能屏蔽单个动态的，点开新的动态仍然会有（广告也都是同一个，b站什么睿智推送啊）
能不能通过插件屏蔽掉所有的，动态首页和up个人空间动态下都有，以防策略不一样提醒一下
![image](https://github.com/the1812/Bilibili-Evolved/assets/72682556/94dc354b-57bb-48d4-8663-65df3623d57a)
顺便问问什么时候插件更新，看帖子修复/适配了不少东西了

**Comment by the1812** [comment]
这个会在新版的简化评论区组件中支持.

更新快的话 08.18 前, 慢的话估计得 08.26~08.27, 这两个月周末几乎都不在家所以进度很慢
